openLuup/timers.lua

```diff
@@ -339,9 +340,10 @@ local function call_timer (fct, timer_type, time, days, data, recurring)
     if u then
       local base = timenow()       -- 2017.07.14
       local increment = math.max (v * multiplier[u], 1)    -- 2017.07.14
-      -- dont_increment parameter added since target is called twice (by start_timer and timer.job)
+      -- dont_increment parameter added since target is called twice (by start_timer and timer.job). Rafale77 reverted. Blocks interval increment
```

Comment:
Reverting this change doesn't have the effect you think it does.  It was there for a reason.

Whilst reverting it it fixes the displayed time, it actually means that the first scheduled run of the interval timer is skipped.  

The displayed time is only wrong before the first run of the timer and, thereafter, is correct.  I'd rather have  correct functioning of the timer and a wrong display than the other way around, so this is very low on the 'fix me' list.